Source/Generators/Number.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import CoreGraphics
+
+private var lastUsedId: Int64 = 0
+
+public class Number {
```

Comment:
@FGoessler I think if there is no need to extend `Generator` it could be a `struct`.